pkg/rigfs/winfsys_test.go

```diff
@@ -0,0 +1,18 @@
+package rigfs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFormatPath(t *testing.T) {
```

Comment:
Maybe add some test for [file namespaces.](https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file#win32-file-namespaces)